apps/super-app/src/services/ServiceSupportDetails.ts

```diff
@@ -86,3 +85,339 @@ const PRO_BONO_SG_DETAIL: Record<LanguageOption, SupportDetail> = {
+    name: 'Aidha',
+    website: 'https://www.aidha.org/',
+    blurbIntro:
+      'Aidha offers courses for helpers in money management, communication, computer and entrepreneurship. Aidha’s mission is to help migrant domestic workers and low-income Singaporean women achieve economic independence through financial education, wealth creation and entrepreneurship. Founded in 2006, Aidha is an award-winning charity with Institution of Public Character (IPC) status. We have 500 students currently enrolled in our programmes.',
+    carouselSection: [],
+    blurbFacebookLinks: {
+      title: '',
+      imageSrc: '',
+      links: [
+        {
+          href: '',
+          text: '',
+        },
+        {
+          href: '',
+          text: '',
+        },
+      ],
+    },
+    ctaButtonText: 'Enrol in free short courses',
+    ctaLink: 'https://www.aidha.org/courses/free-short-courses/',
+    ctaIcon: '',
+  },
+};
+
+const HOME_DETAIL: Record<LanguageOption, SupportDetail> = {
+  en: {
+    logo: 'https://images.squarespace-cdn.com/content/v1/5a12725612abd96b9c737354/1512006074840-DDTG0JDU6HZ72GX7KSO9/HOME+Logo.jpg?format=1500w',
+    name: 'HOME',
+    website: 'https://www.home.org.sg/',
+    blurbIntro:
+      'Humanitarian Organization for Migration Economics [“H.O.M.E.”] is a Singapore-based charity founded in 2004 by the late Ms Bridget Tan. HOME is dedicated to empowering and supporting migrant workers who experience abuse and exploitation. We provide immediate crisis intervention as well as long-term support such as education and training programmes to achieve our goal of enhancing the wellbeing and employment prospects of migrant workers. HOME also promotes the diverse voices of migrant workers in Singapore through outreach projects, and public education. Three pillars guide us: Welfare, Empowerment and Advocacy.',
+    carouselSection: [],
+    blurbFacebookLinks: {
+      title: '',
+      imageSrc: '',
+      links: [
+        {
+          href: '',
+          text: '',
+        },
+        {
+          href: '',
+          text: '',
+        },
+      ],
+    },
+    ctaButtonText: '+65 9787 3122',
+    ctaLink: 'https://wa.me/6597873122',
+    ctaIcon: '/images/whatsapp-icon.svg',
+  },
+};
+
+const CDE_DETAIL: Record<LanguageOption, SupportDetail> = {
+  en: {
+    logo: '/images/cde-logo.png',
+    name: 'Centre for Domestic Employees (CDE)',
+    website: 'https://www.cde.org.sg/wps/portal/cde/home/',
+    blurbIntro:
+      'The Centre for Domestic Employees (CDE) promote the fair treatment of all domestic employees in Singapore through issues resolution, humanitarian aid, social integration and support. We can support you in up-skilling and emotional resilience',
+    carouselSection: [],
+    blurbFacebookLinks: {
+      title: 'Find us on Facebook',
+      imageSrc: '/images/facebook-icon.svg',
+      links: [
+        {
+          href: 'www.facebook.com/cde.singapore',
```

Comment:
```suggestion
          href: 'https://www.facebook.com/cde.singapore',
```
otherwise, the `href` would be treated as a relative instead of an absolute path!